docs/tutorials/t-05.md

```diff
@@ -0,0 +1,124 @@
+# Algo Builder Web Support for Multisignature Transaction
```

Comment:
we already have tutorial 5-7, but we don't have it here. Please move this tutorial to `t-08`.